src/Hyperion/Main.hs

```diff
@@ -123,10 +130,21 @@ options = do
          (Options.switch
             (Options.long "raw" <>
              Options.help "Include raw measurement data in report."))
+     configMonoidExtraMetadata <-
+       many
+         (Options.option
+            toTup
+            (Options.long "arg" <>
+             Options.help "Include raw measurement data in report."))
      -- TODO allow setting this from CLI.
      pure ConfigMonoid{..}
   where
     configMonoidSamplingStrategy = First Nothing
+    toTup = do
+      txt <- Text.pack <$> Options.str
+      case Text.splitOn ":" txt of
+        [x,y] -> pure (x,y)
+        _ -> mzero
```

Comment:
The usage produced by `optparse-applicative` only says `--arg ARG`. Need to be more informative if the splitting failed.